Added fr_FR.lang

CornetPanique86 opened the pull request:
Added French language translation to AxolotlClient

CornetPanique86 committed:
Added fr_FR.lang

French language translation

moehreag reviewed (changes requested):
Remove the colons (:) and it's fine!

moehreag committed:
remove some colons that didn't belong there

- Replace »À« la with »Sur« since it's used for the name of the option (consistency), if my choice is wrong please let me know, I'm no french native..

moehreag committed:
Do it as you said: »À la« instead of »Sur«.
